api/topology/src/main/java/quarks/topology/services/ApplicationService.java

```diff
@@ -63,4 +65,11 @@ Licensed to the Apache Software Foundation (ASF) under one
      * @see ApplicationServiceMXBean
      */
     void registerTopology(String applicationName, BiConsumer<Topology, JsonObject> builder);
+    
```

Comment:
I'm solving a concrete use case, where the device has a number of pre-installed applications but not all are running. Remotely an application can be enabled, for example when a customer pays for a specific feature/service, the device provider turns on that application and presumably events from the device start flowing back to a central system to provide the service to the customer.

If there are other use cases that need to be supported additional functionality can be added, maybe the discussion could move out of this pull request.